src/components/Avatar/Avatar.test.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { Avatar, AvatarProps } from './';
+
+const DEFAULT_PROPS: AvatarProps = {
+  size: 'regular',
+  type: 'circle'
+};
+
+beforeEach(() => {
+  jest.resetAllMocks();
+});
+
+describe('<Avatar />', () => {
+  test('should have only avatar with regular size and circle shape', () => {
```

Comment:
Can you change the names of these to include some context, i.e.

```
describe 'when using default props'
    test 'should be regular size'
    test 'should be circle shape'

describe 'when status type is "active"'
    test 'should show active badge'
```

Reply:
updated the test cases